### PR TITLE
Add permissions and ownership instructions

### PR DESCRIPTION
**Pull request by tankerkiller125**
This pull request addresses #62

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -76,6 +76,18 @@ www.example.com {
...
+During installation Flarum may request that you set some folders to permission level `755`. If after completing these steps Flarum continues to request that you change the permissions you may need to check that your files are owned by the correct group and user. 
+
+By default in most linux distributions `www-data` is the group and user that PHP and the web server operate under. You can change the folder ownership in most linux operating systems by running `chown -R www-data:www-data foldername/`. 
+
+::: warning Environments may vary
+Your environment may vary from the documentation provided, please consult your web server configuration or web hosting provider for the proper user and group that PHP and the web server operate under.
+:::
+
+::: danger Never use permission 777
+You should never set any folder or file to permission level `777` this permission level allows anyone to access the content of the folder and file regardless of user or group.
```
Comment (franzliedke) [inline docs/install.md]: `777, as this`? Or start a new sentence there?

**Comment by franzliedke** [comment]
Generally, I like this. Leaving for @jordanjay29 to merge, though, as I remember seeing you discuss this topic in chat.

That said, we should also consider making the permission checks more intelligent in the installer - that would be even cooler than thorough documentation on this part. :grinning:

File: docs/install.md (hunk 1 above)
Comment (tankerkiller125) [reply docs/install.md]: Thanks for pointing that out, I read through this multiple times but usually at night so I probably was tired and missed it, I'll fix that tonight after work.

File: docs/install.md [hunk 2]
```diff
@@ -76,6 +76,18 @@ www.example.com {
     gzip
 }
 ```
+## Folder Ownership
+During installation Flarum may request that you set some folders to permission level `755`. If after completing these steps Flarum continues to request that you change the permissions you may need to check that your files are owned by the correct group and user.
```
Comment (jordanjay29) [inline docs/install.md]:
```suggestion
During installation, Flarum may request that you set some folders to permission level `755`. If after completing these steps, Flarum continues to request that you change the permissions you may need to check that your files are owned by the correct group and user. 
```

File: docs/install.md [hunk 3]
```diff
@@ -76,6 +76,18 @@ www.example.com {
     gzip
 }
 ```
+## Folder Ownership
+During installation Flarum may request that you set some folders to permission level `755`. If after completing these steps Flarum continues to request that you change the permissions you may need to check that your files are owned by the correct group and user. 
+
+By default in most linux distributions `www-data` is the group and user that PHP and the web server operate under. You can change the folder ownership in most linux operating systems by running `chown -R www-data:www-data foldername/`.
```
Comment (jordanjay29) [inline docs/install.md]:
```suggestion
By default in most linux distributions, `www-data` is the group and user that both PHP and the web server operate under. You can change the folder ownership in most linux operating systems by running `chown -R www-data:www-data foldername/`. 
```

File: docs/install.md (hunk 1 above)
Comment (jordanjay29) [reply docs/install.md]:
```suggestion
You should never set any folder or file to permission level `777`, as this permission level allows anyone to access the content of the folder and file regardless of user or group. 
```

This one seemed to get missed. Fix and I'll approve the PR.

Comment (tankerkiller125) [reply docs/install.md]: @jordanjay29 The intent was to fix it, unfortunately I got very busy at work and had to do some work from home so I didn't have time to do it immediately.

**Pull request merged**